### ISSUE TITLE
Integrate with Jetbrains on Linux

### ISSUE THREAD

**Issue by grant0417** [withfig/fig#1691]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my feature

### Feature Details

We are currently blocked by this [issue](https://youtrack.jetbrains.com/issue/JBR-2460/Wrong-position-of-input-window-and-no-input-preview-with-fcitx-and-ubuntu-1304) which prevents IME from properly reporting its location in Jetbrains IDEs
